src/cdef.rs

```diff
@@ -1,11 +1,27 @@
+use crate::include::common::attributes::clz;
+use crate::include::common::bitdepth::AsPrimitive;
+use crate::include::common::bitdepth::BitDepth;
 use crate::include::common::bitdepth::DynPixel;
 use crate::include::common::bitdepth::LeftPixelRow2px;
 use crate::include::common::intops::apply_sign;
+use crate::include::common::intops::iclip;
+use crate::include::common::intops::ulog2;
+use crate::src::tables::dav1d_cdef_directions;
+
 use libc::ptrdiff_t;
 use std::cmp;
 use std::ffi::c_int;
 use std::ffi::c_uint;
 
+#[cfg(feature = "asm")]
+use cfg_if::cfg_if;
+
+#[cfg(feature = "asm")]
+use crate::src::cpu::{rav1d_get_cpu_flags, CpuFlags};
+
+#[cfg(all(feature = "asm", any(target_arch = "arm", target_arch = "aarch64"),))]
```

Comment:
```suggestion
#[cfg(feature = "asm")]
```

See below for the other import removed.